### ISSUE TITLE
core service up? No internet with or without filtering toggle

### ISSUE THREAD

**Issue by working-name** [safing/portmaster#1089]
**Pre-Submit Checklist**:

- Check applicable sources for existing issues:
  - [Windows Known Issues](https://docs.safing.io/portmaster/install/windows#known-issues)
  - [Linux Known Issues](https://docs.safing.io/portmaster/install/linux#compatibility)
  - [Github Issues](https://github.com/safing/portmaster/issues?q=is%3Aissue+label%3Abug)

**What happened**:
This version is pretty useless, it doesn't respond to any DNS queries and seems to block connection attempts from all apps after it runs for a while. I thought it was my internet but after restarting the router I tested things out from another computer that doesn't have portmaster and all was peachy.

In addition, lately it seems it can't even resolve the IPs it's trying to filter. Instead of domain names I get a long list of random IPs some app want to connect to. Not as useful.

Yet another issue is that it randomly thinks it's offline. I don't quite get the check, last time I opened an issue about it it seemed to try to get to 1.1.1.1 but haven't looked into what it's doing now.

I swapped system DNS servers from ISP to google, then cloudflare, then dns.watch. Does not make a difference.

Right now the only reason I can access github.com is because I killed portmaster-core process. Privacy Filter Module is already off but still no go.

**What did you expect to happen?**:
Work.

**How did you reproduce it?**:
Start portmaster, or simply reboot computer. Give it some time and it'll stop working.

**Debug Information**:
There's no "Copy Debug Information" on the settings page itself. Used the tools icon while on the settings page. No clue if that's what you mean here or not.

<details>
<summary>Version 1.0.6</summary>

```
Portmaster
version 1.0.6

commit tags/v1.0.6-0-g0ee078d911665739696ed89e2582768e0ba6b864
built with go1.19 (gc) windows/amd64
  using options main.go
  by user@docker
  on 30.01.2023

Licensed under the AGPLv3 license.
The source code is available here: https://github.com/safing/portmaster
```

</details>

<details>
<summary>Platform: Microsoft Windows 11 Pro 10.0.22000 Build 22000</summary>

```
System: Microsoft Windows 11 Pro windows (Standalone Workstation) 10.0.22000 Build 22000
Kernel: 10.0.22000 Build 22000 x86_64

```

</details>

<details>
<summary>Status: Trusted</summary>

```
ActiveSecurityLevel:   Trusted
SelectedSecurityLevel: Off
ThreatMitigationLevel: Trusted
CaptivePortal:         
OnlineStatus:          Online
```

</details>

<details>
<summary>Config: 16</summary>

```
core/devMode: true
core/expertiseLevel: developer
core/metrics/instance: [redacted]
core/releaseChannel: beta
core/releaseLevel: beta
core/useSystemNotifications: false
dns/nameserverRetryRate: 300
dns/nameservers: [redacted]
dns/noAssignedNameservers: 6
filter/askTimeout: 30
filter/askWithSystemNotifications: false
filter/blockInbound: 6
filter/defaultAction: ask
filter/enable: false
filter/endpoints: [redacted]
filter/serviceEndpoints: [redacted]
```

</details>

<details>
<summary>Resolvers: 2/2</summary>

```
84.200.69.80 (dns://84.200.69.80:53#system)
  dns://84.200.69.80:53#system
  Failing: false

84.200.70.40 (dns://84.200.70.40:53#system)
  dns://84.200.70.40:53#system
  Failing: false
```

</details>

<details>
<summary>SPN: disabled (module disabled)</summary>

```
HomeHubID:    
HomeHubName:  
HomeHubIP:    
Transport:    
---
Client:       true
PublicHub:    false
HubHasIPv4:   false
HubHasIPv6:   false
```

</details>

<details>
<summary>Compatibility: WFP State (17)</summary>

```
Edge traversal Teredo Authorization Sublayer   SubLayer   Edge traversal Teredo Authorization Sublayer                                 {7b6b11f6-cbb5-433c-ae06-6a4f0076e49e}
IPxlat Forward IPv4 filter                     Callout    Filters forwarded IPv4 packets into synthetic IPv6 packets                   {b255c296-7e0c-4115-95f3-b7f24a8a1162}   [no provider key]   FWPM_LAYER_IPFORWARD_V4
IPxlat Forward IPv4 sub layer                  SubLayer   Sub layer for filtering forwarded IPv4 packets into synthetic IPv6 packets   {4351e497-5d8b-46bc-86d9-abccdb868d6d}
IPxlat Inbound IPv6 filter                     Callout    Filters incoming IPv6 packets into synthetic IPv4 packets                    {93bb703d-0502-42e2-8e30-a14576e5085d}   [no provider key]   FWPM_LAYER_INBOUND_IPPACKET_V6
IPxlat Inbound IPv6 sub layer                  SubLayer   Sub layer for filtering incoming IPv6 packets into synthetic IPv4 packets    {dfb035ca-c2a7-4684-97b6-4dbc57c63590}
IPxlat Outbound IPv4 filter                    Callout    Filters outgoing IPv4 packets into synthetic IPv6 packets                    {66d52657-1979-4e58-b3f7-4756434c4880}   [no provider key]   FWPM_LAYER_OUTBOUND_IPPACKET_V4
IPxlat Outbound IPv4 sub layer                 SubLayer   Sub layer for filtering outgoing IPv4 packets into synthetic IPv6 packets    {d3e70856-fc90-4c0a-b9b2-a6f73e20b5cc}
PortmasterInboundV4Callout                     Callout    This callout is used by the Portmaster to intercept inbound IPv4 traffic.    {05c55149-4732-4857-8d10-f178f3a06f8c}   [no provider key]   FWPM_LAYER_INBOUND_IPPACKET_V4
PortmasterInboundV4Filter                      Filter     This filter is used by the Portmaster to intercept inbound IPv4 traffic.     {72645d9c-fe08-4dcb-885e-eeeecc8e5347}   [no provider key]   FWPM_LAYER_INBOUND_IPPACKET_V4   {a87fb472-fc68-4805-8559-c6ae774773e0}
PortmasterInboundV6Callout                     Callout    This callout is used by the Portmaster to intercept inbound IPv6 traffic.    {ceff1df7-2baa-44c5-a6e5-73a95849bcff}   [no provider key]   FWPM_LAYER_INBOUND_IPPACKET_V6
PortmasterInboundV6Filter                      Filter     This filter is used by the Portmaster to intercept inbound IPv6 traffic.     {49e85738-b724-459d-a94a-6130859e89df}   [no provider key]   FWPM_LAYER_INBOUND_IPPACKET_V6   {a87fb472-fc68-4805-8559-c6ae774773e0}
PortmasterOutboundV4Callout                    Callout    This callout is used by the Portmaster to intercept outbound IPv4 traffic.   {41162b9e-8473-4b88-a5eb-04cf1d276b06}   [no provider key]   FWPM_LAYER_OUTBOUND_IPPACKET_V4
PortmasterOutboundV4Filter                     Filter     This filter is used by the Portmaster to intercept outbound IPv4 traffic.    {cd7a2fa1-5e4e-4216-9ee7-864674080e17}   [no provider key]   FWPM_LAYER_OUTBOUND_IPPACKET_V4   {a87fb472-fc68-4805-8559-c6ae774773e0}
PortmasterOutboundV6Callout                    Callout    This callout is used by the Portmaster to intercept outbound IPv6 traffic.   {32bad112-6af4-4109-809b-c07570ba01b4}   [no provider key]   FWPM_LAYER_OUTBOUND_IPPACKET_V6
PortmasterOutboundV6Filter                     Filter     This filter is used by the Portmaster to intercept outbound IPv6 traffic.    {7471f481-a753-425b-9f99-3ee25b575cf4}   [no provider key]   FWPM_LAYER_OUTBOUND_IPPACKET_V6   {a87fb472-fc68-4805-8559-c6ae774773e0}
PortmasterSublayer                             SubLayer   The Portmaster sublayer holds all it's filters.                              {a87fb472-fc68-4805-8559-c6ae774773e0}
Teredo socket option opt out block filter      Filter     [no description]                                                             {27034f3a-707b-44fb-a0c8-ccb0d561aabb}   {b74aab64-fab5-4f64-ac42-89296dd893ff}   FWPM_LAYER_ALE_AUTH_RECV_ACCEPT_V6   {7b6b11f6-cbb5-433c-ae06-6a4f0076e49e}

```

</details>

<details>
<summary>No Module Error</summary>

</details>

<details>
<summary>Unexpected Logs</summary>

```
230208 06:42:59.297 nameserver:286 > WARN 012 nameserver: failed to resolve client.dropbox.com.A: all 2 query-compliant resolvers failed, last error: read udp 10.0.0.206:49118->84.200.70.40:53: i/o timeout
230208 06:43:06.356 ine-status:534 > WARN 014 netenv: dns query check failed: all 2 query-compliant resolvers failed, last error: read udp 10.0.0.206:13941->84.200.70.40:53: i/o timeout
230208 06:43:46.875 ine-status:534 > WARN 016 netenv: dns query check failed: all 2 query-compliant resolvers failed, last error: read udp 10.0.0.206:20875->84.200.70.40:53: i/o timeout
230208 06:44:05.932 ine-status:534 > WARN 018 netenv: dns query check failed: all 2 query-compliant resolvers failed, last error: read udp 10.0.0.206:20577->84.200.70.40:53: i/o timeout
230208 06:45:02.670 nameserver:286 > WARN 020 nameserver: failed to resolve update.googleapis.com.A: all 2 query-compliant resolvers failed, last error: read udp 10.0.0.206:12027->84.200.70.40:53: i/o timeout
230208 06:45:10.750 nameserver:286 > WARN 022 nameserver: failed to resolve www.youtube.com.A: all 2 query-compliant resolvers failed, last error: read udp 10.0.0.206:49172->84.200.70.40:53: i/o timeout
230208 06:45:11.532 nameserver:286 > WARN 024 nameserver: failed to resolve data.amazon.com.A: all 2 query-compliant resolvers failed, last error: read udp 10.0.0.206:63773->84.200.70.40:53: i/o timeout
230208 06:45:24.846 nameserver:286 > WARN 026 nameserver: failed to resolve ctldl.windowsupdate.com.A: all 2 query-compliant resolvers failed, last error: read udp 10.0.0.206:24099->84.200.70.40:53: i/o timeout
230208 06:46:01.392 nameserver:286 > WARN 028 nameserver: failed to resolve play.google.com.A: all 2 query-compliant resolvers failed, last error: dial udp :54787->84.200.70.40:53: bind: An attempt was made to access a socket in a way forbidden by its access permissions.
230208 06:47:14.729 nameserver:286 > WARN 030 nameserver: failed to resolve www.youtube.com.A: all 2 query-compliant resolvers failed, last error: read udp 10.0.0.206:22029->84.200.70.40:53: i/o timeout
230208 06:47:51.442 CURRENT TIME
```

</details>

<details>
<summary>Updates: beta (8/31)</summary>

```
Active:
all/intel/geoip/geoipv4.mmdb.gz: 20230102.9.32
all/intel/geoip/geoipv6.mmdb.gz: 20230102.13.14
all/intel/portmaster/notifications.yaml: 20221101.12.34
all/ui/modules/assets.zip: 0.3.1
all/ui/modules/portmaster.zip: 0.4.5
windows_amd64/core/portmaster-core.exe: 1.0.6
windows_amd64/kext/portmaster-kext.sys: 1.0.17
windows_amd64/start/portmaster-start.exe: 1.0.2

Selected:
all/intel/geoip/geoipv4.mmdb.gz: 20230102.9.32
all/intel/geoip/geoipv4.mmdb: 20230102.9.32
all/intel/geoip/geoipv6.mmdb.gz: 20230102.13.14
all/intel/geoip/geoipv6.mmdb: 20230102.13.14
all/intel/lists/base.dsdl: 20221231.23.0
all/intel/lists/index.dsd: 2022.6.7
all/intel/lists/intermediate.dsdl: 20221231.23.0
all/intel/lists/urgent.dsdl: 20230127.10.0
all/intel/portmaster/notifications.json: 20211214.14.13
all/intel/portmaster/notifications.yaml: 20221101.12.34
all/intel/spn/main-intel.json: 20220113.8.47
all/intel/spn/main-intel.yaml: 20230207.13.47
all/ui/modules/assets.zip: 0.3.1
all/ui/modules/base.zip: 0.2.11
all/ui/modules/console.zip: 0.1.11
all/ui/modules/monitor.zip: 0.2.4
all/ui/modules/portmaster.zip: 0.4.5
all/ui/modules/profilemgr.zip: 0.1.7
all/ui/modules/settings.zip: 0.1.8
windows_amd64/app/portmaster-app.zip: 0.2.5
windows_amd64/core/portmaster-core.exe: 1.0.6
windows_amd64/hub/spn-hub.exe: 0.6.1
windows_amd64/jess/jess.exe: 0.3.1
windows_amd64/kext/portmaster-kext.dll: 1.0.14
windows_amd64/kext/portmaster-kext.pdb: 1.0.17
windows_amd64/kext/portmaster-kext.sys: 1.0.17
windows_amd64/notifier/portmaster-notifier.exe: 0.3.5
windows_amd64/notifier/portmaster-snoretoast.exe: 0.6.0
windows_amd64/notifier/portmaster-wintoast.dll: 0.1.4
windows_amd64/packages/portmaster-installer.exe: 1.0.0
windows_amd64/start/portmaster-start.exe: 1.0.2
```

</details>

<details>
<summary>Goroutine Stack</summary>

```
goroutine profile: total 78
33 @ 0x4bd216 0x4ccd5c 0x845446 0x845a36 0x4ea1a1
#	0x845445	github.com/safing/portbase/api.(*DatabaseAPI).processSub+0x225	/home/user/git/safing/portbase/api/database.go:419
#	0x845a35	github.com/safing/portbase/api.(*DatabaseAPI).handleQsub+0xd5	/home/user/git/safing/portbase/api/database.go:482

3 @ 0x4bd216 0x4b32d7 0x4e3ae9 0x526ed2 0x5281a5 0x52902b 0x5b13c9 0x5c0c25 0x761b31 0x5e585f 0x5e59bd 0x763078 0x767605 0x4ea1a1
#	0x4e3ae8	internal/poll.runtime_pollWait+0x88	/usr/local/go/src/runtime/netpoll.go:305
#	0x526ed1	internal/poll.(*pollDesc).wait+0x31	/usr/local/go/src/internal/poll/fd_poll_runtime.go:84
#	0x5281a4	internal/poll.execIO+0xe4		/usr/local/go/src/internal/poll/fd_windows.go:175
#	0x52902a	internal/poll.(*FD).Read+0x26a		/usr/local/go/src/internal/poll/fd_windows.go:441
#	0x5b13c8	net.(*netFD).Read+0x28			/usr/local/go/src/net/fd_posix.go:55
#	0x5c0c24	net.(*conn).Read+0x44			/usr/local/go/src/net/net.go:183
#	0x761b30	net/http.(*connReader).Read+0x170	/usr/local/go/src/net/http/server.go:786
#	0x5e585e	bufio.(*Reader).fill+0xfe		/usr/local/go/src/bufio/bufio.go:106
#	0x5e59bc	bufio.(*Reader).Peek+0x5c		/usr/local/go/src/bufio/bufio.go:144
#	0x763077	net/http.(*conn).readRequest+0x1f7	/usr/local/go/src/net/http/server.go:991
#	0x767604	net/http.(*conn).serve+0x344		/usr/local/go/src/net/http/server.go:1916

2 @ 0x4bd216 0x4ccd5c 0x7f8105 0x8079d3 0x8073b6 0x4ea1a1
#	0x7f8104	github.com/safing/portbase/database.(*Interface).DelayedCacheWriter+0x184	/home/user/git/safing/portbase/database/interface_cache.go:34
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92			/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135		/home/user/git/safing/portbase/modules/worker.go:83

2 @ 0x4bd216 0x4ccd5c 0x82fce6 0x8079d3 0x8073b6 0x4ea1a1
#	0x82fce5	github.com/safing/portbase/rng.(*Feeder).run+0x165			/home/user/git/safing/portbase/rng/entropy.go:117
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

2 @ 0x4bd216 0x4ccd5c 0x845446 0x845072 0x4ea1a1
#	0x845445	github.com/safing/portbase/api.(*DatabaseAPI).processSub+0x225	/home/user/git/safing/portbase/api/database.go:419
#	0x845071	github.com/safing/portbase/api.(*DatabaseAPI).handleSub+0x91	/home/user/git/safing/portbase/api/database.go:391

1 @ 0x4842ea 0x4e6c29 0x4e68b0 0x5024b4 0x4fc1cf 0x4fc1c2 0x528f71 0x53301e 0x533016 0x5e7df6 0x85c1c5 0x4ea1a1
#	0x4e6c28	syscall.SyscallN+0x108					/usr/local/go/src/runtime/syscall_windows.go:557
#	0x4e68af	syscall.Syscall6+0x4f					/usr/local/go/src/runtime/syscall_windows.go:501
#	0x5024b3	syscall.readFile+0x93					/usr/local/go/src/syscall/zsyscall_windows.go:1024
#	0x4fc1ce	syscall.ReadFile+0x2e					/usr/local/go/src/syscall/syscall_windows.go:400
#	0x4fc1c1	syscall.Read+0x21					/usr/local/go/src/syscall/syscall_windows.go:379
#	0x528f70	internal/poll.(*FD).Read+0x1b0				/usr/local/go/src/internal/poll/fd_windows.go:427
#	0x53301d	os.(*File).read+0x5d					/usr/local/go/src/os/file_posix.go:31
#	0x533015	os.(*File).Read+0x55					/usr/local/go/src/os/file.go:118
#	0x5e7df5	bufio.(*Scanner).Scan+0x875				/usr/local/go/src/bufio/scan.go:214
#	0x85c1c4	github.com/safing/portbase/run.inputSignals+0x84	/home/user/git/safing/portbase/run/main.go:126

1 @ 0x4842ea 0x4e6c29 0x4e6938 0x5d2873 0xcef145 0xcef20b 0xcec6c5 0xceb67f 0x4ea1a1
#	0x4e6c28	syscall.SyscallN+0x108										/usr/local/go/src/runtime/syscall_windows.go:557
#	0x4e6937	syscall.Syscall9+0x77										/usr/local/go/src/runtime/syscall_windows.go:507
#	0x5d2872	golang.org/x/sys/windows.DeviceIoControl+0xd2							/home/user/go/pkg/mod/golang.org/x/sys@v0.4.0/windows/zsyscall_windows.go:1797
#	0xcef144	github.com/safing/portmaster/firewall/interception/windowskext.deviceIOControlAsync+0xa4	/home/user/git/safing/portmaster/firewall/interception/windowskext/syscall.go:47
#	0xcef20a	github.com/safing/portmaster/firewall/interception/windowskext.deviceIOControl+0x2a		/home/user/git/safing/portmaster/firewall/interception/windowskext/syscall.go:62
#	0xcec6c4	github.com/safing/portmaster/firewall/interception/windowskext.RecvVerdictRequest+0x144		/home/user/git/safing/portmaster/firewall/interception/windowskext/kext.go:137
#	0xceb67e	github.com/safing/portmaster/firewall/interception/windowskext.Handler+0x7e			/home/user/git/safing/portmaster/firewall/interception/windowskext/handler.go:84

1 @ 0x48c43c 0x4e582f 0x85ba79 0x4ea1a1
#	0x4e582e	os/signal.signal_recv+0x2e	/usr/local/go/src/runtime/sigqueue.go:152
#	0x85ba78	os/signal.loop+0x18		/usr/local/go/src/os/signal/signal_unix.go:23

1 @ 0x4afd76 0x4e36e5 0x80f395 0x80f1ad 0x80c14b 0x819d25 0xce8ecf 0x848b23 0x848369 0x84d4f3 0x84c0a9 0x8079d3 0x806f77 0x84c02d 0x76b9ac 0x7678c7 0x4ea1a1
#	0x4e36e4	runtime/pprof.runtime_goroutineProfileWithLabels+0x24			/usr/local/go/src/runtime/mprof.go:846
#	0x80f394	runtime/pprof.writeRuntimeProfile+0xb4					/usr/local/go/src/runtime/pprof/pprof.go:723
#	0x80f1ac	runtime/pprof.writeGoroutine+0x4c					/usr/local/go/src/runtime/pprof/pprof.go:683
#	0x80c14a	runtime/pprof.(*Profile).WriteTo+0x14a					/usr/local/go/src/runtime/pprof/pprof.go:330
#	0x819d24	github.com/safing/portbase/utils/debug.(*Info).AddGoroutineStack+0x64	/home/user/git/safing/portbase/utils/debug/debug.go:133
#	0xce8ece	github.com/safing/portmaster/core.debugInfo+0x12e			/home/user/git/safing/portmaster/core/api.go:150
#	0x848b22	github.com/safing/portbase/api.(*Endpoint).ServeHTTP+0x742		/home/user/git/safing/portbase/api/endpoints.go:442
#	0x848368	github.com/safing/portbase/api.(*endpointHandler).ServeHTTP+0x48	/home/user/git/safing/portbase/api/endpoints.go:358
#	0x84d4f2	github.com/safing/portbase/api.(*mainHandler).handle+0x1432		/home/user/git/safing/portbase/api/router.go:282
#	0x84c0a8	github.com/safing/portbase/api.(*mainHandler).ServeHTTP.func1+0x28	/home/user/git/safing/portbase/api/router.go:84
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x806f76	github.com/safing/portbase/modules.(*Module).RunWorker+0x76		/home/user/git/safing/portbase/modules/worker.go:52
#	0x84c02c	github.com/safing/portbase/api.(*mainHandler).ServeHTTP+0x6c		/home/user/git/safing/portbase/api/router.go:83
#	0x76b9ab	net/http.serverHandler.ServeHTTP+0x30b					/usr/local/go/src/net/http/server.go:2947
#	0x7678c6	net/http.(*conn).serve+0x606						/usr/local/go/src/net/http/server.go:1991

1 @ 0x4bd216 0x4869bb 0x4864b8 0xaae3ba 0x8079d3 0x8073b6 0x4ea1a1
#	0xaae3b9	github.com/safing/portmaster/resolver.listenToMDNS+0x619		/home/user/git/safing/portmaster/resolver/resolver-mdns.go:146
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4b32d7 0x4e3ae9 0x526ed2 0x5281a5 0x52902b 0x5b13c9 0x5c0c25 0x5e585f 0x5e59bd 0x83314c 0x83527b 0x8362ac 0x8368b9 0x8428f5 0x8079d3 0x806f77 0x806d6a 0x4ea1a1
#	0x4e3ae8	internal/poll.runtime_pollWait+0x88					/usr/local/go/src/runtime/netpoll.go:305
#	0x526ed1	internal/poll.(*pollDesc).wait+0x31					/usr/local/go/src/internal/poll/fd_poll_runtime.go:84
#	0x5281a4	internal/poll.execIO+0xe4						/usr/local/go/src/internal/poll/fd_windows.go:175
#	0x52902a	internal/poll.(*FD).Read+0x26a						/usr/local/go/src/internal/poll/fd_windows.go:441
#	0x5b13c8	net.(*netFD).Read+0x28							/usr/local/go/src/net/fd_posix.go:55
#	0x5c0c24	net.(*conn).Read+0x44							/usr/local/go/src/net/net.go:183
#	0x5e585e	bufio.(*Reader).fill+0xfe						/usr/local/go/src/bufio/bufio.go:106
#	0x5e59bc	bufio.(*Reader).Peek+0x5c						/usr/local/go/src/bufio/bufio.go:144
#	0x83314b	github.com/gorilla/websocket.(*Conn).read+0x2b				/home/user/go/pkg/mod/github.com/gorilla/websocket@v1.5.0/conn.go:371
#	0x83527a	github.com/gorilla/websocket.(*Conn).advanceFrame+0x7a			/home/user/go/pkg/mod/github.com/gorilla/websocket@v1.5.0/conn.go:809
#	0x8362ab	github.com/gorilla/websocket.(*Conn).NextReader+0xcb			/home/user/go/pkg/mod/github.com/gorilla/websocket@v1.5.0/conn.go:1009
#	0x8368b8	github.com/gorilla/websocket.(*Conn).ReadMessage+0x18			/home/user/go/pkg/mod/github.com/gorilla/websocket@v1.5.0/conn.go:1093
#	0x8428f4	github.com/safing/portbase/api.(*DatabaseAPI).handler+0x94		/home/user/git/safing/portbase/api/database.go:150
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x806f76	github.com/safing/portbase/modules.(*Module).RunWorker+0x76		/home/user/git/safing/portbase/modules/worker.go:52
#	0x806d69	github.com/safing/portbase/modules.(*Module).StartWorker.func1+0x49	/home/user/git/safing/portbase/modules/worker.go:27

1 @ 0x4bd216 0x4b32d7 0x4e3ae9 0x526ed2 0x5281a5 0x52902b 0x5b13c9 0x5c0c25 0x76165f 0x4ea1a1
#	0x4e3ae8	internal/poll.runtime_pollWait+0x88		/usr/local/go/src/runtime/netpoll.go:305
#	0x526ed1	internal/poll.(*pollDesc).wait+0x31		/usr/local/go/src/internal/poll/fd_poll_runtime.go:84
#	0x5281a4	internal/poll.execIO+0xe4			/usr/local/go/src/internal/poll/fd_windows.go:175
#	0x52902a	internal/poll.(*FD).Read+0x26a			/usr/local/go/src/internal/poll/fd_windows.go:441
#	0x5b13c8	net.(*netFD).Read+0x28				/usr/local/go/src/net/fd_posix.go:55
#	0x5c0c24	net.(*conn).Read+0x44				/usr/local/go/src/net/net.go:183
#	0x76165e	net/http.(*connReader).backgroundRead+0x3e	/usr/local/go/src/net/http/server.go:678

1 @ 0x4bd216 0x4b32d7 0x4e3ae9 0x526ed2 0x5281a5 0x52902b 0x5b13c9 0x5c0c25 0xab0285 0xaae50e 0x8079d3 0x8073b6 0x4ea1a1
#	0x4e3ae8	internal/poll.runtime_pollWait+0x88					/usr/local/go/src/runtime/netpoll.go:305
#	0x526ed1	internal/poll.(*pollDesc).wait+0x31					/usr/local/go/src/internal/poll/fd_poll_runtime.go:84
#	0x5281a4	internal/poll.execIO+0xe4						/usr/local/go/src/internal/poll/fd_windows.go:175
#	0x52902a	internal/poll.(*FD).Read+0x26a						/usr/local/go/src/internal/poll/fd_windows.go:441
#	0x5b13c8	net.(*netFD).Read+0x28							/usr/local/go/src/net/fd_posix.go:55
#	0x5c0c24	net.(*conn).Read+0x44							/usr/local/go/src/net/net.go:183
#	0xab0284	github.com/safing/portmaster/resolver.listenForDNSPackets+0x84		/home/user/git/safing/portmaster/resolver/resolver-mdns.go:342
#	0xaae50d	github.com/safing/portmaster/resolver.listenToMDNS.func7+0x2d		/home/user/git/safing/portmaster/resolver/resolver-mdns.go:130
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4b32d7 0x4e3ae9 0x526ed2 0x5281a5 0x52902b 0x5b13c9 0x5c0c25 0xab0285 0xaae5ce 0x8079d3 0x8073b6 0x4ea1a1
#	0x4e3ae8	internal/poll.runtime_pollWait+0x88					/usr/local/go/src/runtime/netpoll.go:305
#	0x526ed1	internal/poll.(*pollDesc).wait+0x31					/usr/local/go/src/internal/poll/fd_poll_runtime.go:84
#	0x5281a4	internal/poll.execIO+0xe4						/usr/local/go/src/internal/poll/fd_windows.go:175
#	0x52902a	internal/poll.(*FD).Read+0x26a						/usr/local/go/src/internal/poll/fd_windows.go:441
#	0x5b13c8	net.(*netFD).Read+0x28							/usr/local/go/src/net/fd_posix.go:55
#	0x5c0c24	net.(*conn).Read+0x44							/usr/local/go/src/net/net.go:183
#	0xab0284	github.com/safing/portmaster/resolver.listenForDNSPackets+0x84		/home/user/git/safing/portmaster/resolver/resolver-mdns.go:342
#	0xaae5cd	github.com/safing/portmaster/resolver.listenToMDNS.func3+0x2d		/home/user/git/safing/portmaster/resolver/resolver-mdns.go:103
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4b32d7 0x4e3ae9 0x526ed2 0x5281a5 0x52902b 0x5b13c9 0x5c0c25 0xab0285 0xaae62e 0x8079d3 0x8073b6 0x4ea1a1
#	0x4e3ae8	internal/poll.runtime_pollWait+0x88					/usr/local/go/src/runtime/netpoll.go:305
#	0x526ed1	internal/poll.(*pollDesc).wait+0x31					/usr/local/go/src/internal/poll/fd_poll_runtime.go:84
#	0x5281a4	internal/poll.execIO+0xe4						/usr/local/go/src/internal/poll/fd_windows.go:175
#	0x52902a	internal/poll.(*FD).Read+0x26a						/usr/local/go/src/internal/poll/fd_windows.go:441
#	0x5b13c8	net.(*netFD).Read+0x28							/usr/local/go/src/net/fd_posix.go:55
#	0x5c0c24	net.(*conn).Read+0x44							/usr/local/go/src/net/net.go:183
#	0xab0284	github.com/safing/portmaster/resolver.listenForDNSPackets+0x84		/home/user/git/safing/portmaster/resolver/resolver-mdns.go:342
#	0xaae62d	github.com/safing/portmaster/resolver.listenToMDNS.func1+0x2d		/home/user/git/safing/portmaster/resolver/resolver-mdns.go:90
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4b32d7 0x4e3ae9 0x526ed2 0x5281a5 0x52a730 0x5b1709 0x5ca125 0x5c8471 0x5c8630 0x89c765 0x88b490 0x888af9 0x88a189 0x88923b 0xd027ab 0x8079d3 0x8073b6 0x4ea1a1
#	0x4e3ae8	internal/poll.runtime_pollWait+0x88					/usr/local/go/src/runtime/netpoll.go:305
#	0x526ed1	internal/poll.(*pollDesc).wait+0x31					/usr/local/go/src/internal/poll/fd_poll_runtime.go:84
#	0x5281a4	internal/poll.execIO+0xe4						/usr/local/go/src/internal/poll/fd_windows.go:175
#	0x52a72f	internal/poll.(*FD).ReadFromInet6+0x14f					/usr/local/go/src/internal/poll/fd_windows.go:639
#	0x5b1708	net.(*netFD).readFromInet6+0x28						/usr/local/go/src/net/fd_posix.go:72
#	0x5ca124	net.(*UDPConn).readFrom+0x84						/usr/local/go/src/net/udpsock_posix.go:59
#	0x5c8470	net.(*UDPConn).readFromUDP+0x30						/usr/local/go/src/net/udpsock.go:149
#	0x5c862f	net.(*UDPConn).ReadFrom+0x4f						/usr/local/go/src/net/udpsock.go:158
#	0x89c764	github.com/miekg/dns.ReadFromSessionUDP+0x24				/home/user/go/pkg/mod/github.com/miekg/dns@v1.1.50/udp_windows.go:19
#	0x88b48f	github.com/miekg/dns.(*Server).readUDP+0x10f				/home/user/go/pkg/mod/github.com/miekg/dns@v1.1.50/server.go:695
#	0x888af8	github.com/miekg/dns.defaultReader.ReadUDP+0x18				/home/user/go/pkg/mod/github.com/miekg/dns@v1.1.50/server.go:174
#	0x88a188	github.com/miekg/dns.(*Server).serveUDP+0x2a8				/home/user/go/pkg/mod/github.com/miekg/dns@v1.1.50/server.go:513
#	0x88923a	github.com/miekg/dns.(*Server).ListenAndServe+0x49a			/home/user/go/pkg/mod/github.com/miekg/dns@v1.1.50/server.go:342
#	0xd027aa	github.com/safing/portmaster/nameserver.startListener.func1+0x24a	/home/user/git/safing/portmaster/nameserver/module.go:147
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4b32d7 0x4e3ae9 0x526ed2 0x5281a5 0x52cf8d 0x52d2f6 0x5b30e5 0x5c79c8 0x5c683d 0x76bf05 0x76bb3d 0x84bdc5 0x8079d3 0x806f77 0x84bf45 0x4ea1a1
#	0x4e3ae8	internal/poll.runtime_pollWait+0x88				/usr/local/go/src/runtime/netpoll.go:305
#	0x526ed1	internal/poll.(*pollDesc).wait+0x31				/usr/local/go/src/internal/poll/fd_poll_runtime.go:84
#	0x5281a4	internal/poll.execIO+0xe4					/usr/local/go/src/internal/poll/fd_windows.go:175
#	0x52cf8c	internal/poll.(*FD).acceptOne+0x6c				/usr/local/go/src/internal/poll/fd_windows.go:942
#	0x52d2f5	internal/poll.(*FD).Accept+0x1d5				/usr/local/go/src/internal/poll/fd_windows.go:976
#	0x5b30e4	net.(*netFD).accept+0x64					/usr/local/go/src/net/fd_windows.go:139
#	0x5c79c7	net.(*TCPListener).accept+0x27					/usr/local/go/src/net/tcpsock_posix.go:142
#	0x5c683c	net.(*TCPListener).Accept+0x3c					/usr/local/go/src/net/tcpsock.go:288
#	0x76bf04	net/http.(*Server).Serve+0x384					/usr/local/go/src/net/http/server.go:3070
#	0x76bb3c	net/http.(*Server).ListenAndServe+0x7c				/usr/local/go/src/net/http/server.go:2999
#	0x84bdc4	github.com/safing/portbase/api.Serve.func1+0x24			/home/user/git/safing/portbase/api/router.go:66
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92	/home/user/git/safing/portbase/modules/worker.go:128
#	0x806f76	github.com/safing/portbase/modules.(*Module).RunWorker+0x76	/home/user/git/safing/portbase/modules/worker.go:52
#	0x84bf44	github.com/safing/portbase/api.Serve+0x164			/home/user/git/safing/portbase/api/router.go:65

1 @ 0x4bd216 0x4ccd5c 0x62d186 0x62cfea 0x4ea1a1
#	0x62d185	github.com/safing/portbase/log.writer+0xe5		/home/user/git/safing/portbase/log/output.go:154
#	0x62cfe9	github.com/safing/portbase/log.writerManager+0x89	/home/user/git/safing/portbase/log/output.go:113

1 @ 0x4bd216 0x4ccd5c 0x7ff266 0x4ea1a1
#	0x7ff265	github.com/safing/portbase/modules.microTaskScheduler+0x2a5	/home/user/git/safing/portbase/modules/microtasks.go:292

1 @ 0x4bd216 0x4ccd5c 0x8067a5 0x4ea1a1
#	0x8067a4	github.com/safing/portbase/modules.taskQueueHandler+0x84	/home/user/git/safing/portbase/modules/tasks.go:468

1 @ 0x4bd216 0x4ccd5c 0x806ab8 0x4ea1a1
#	0x806ab7	github.com/safing/portbase/modules.taskScheduleHandler+0xb7	/home/user/git/safing/portbase/modules/tasks.go:518

1 @ 0x4bd216 0x4ccd5c 0x8300ce 0x8079d3 0x8073b6 0x4ea1a1
#	0x8300cd	github.com/safing/portbase/rng.fullFeeder+0x8d				/home/user/git/safing/portbase/rng/fullfeed.go:24
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0x830b05 0x8079d3 0x8073b6 0x4ea1a1
#	0x830b04	github.com/safing/portbase/rng.osFeeder+0x184				/home/user/git/safing/portbase/rng/osfeeder.go:26
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0x831465 0x8079d3 0x8073b6 0x4ea1a1
#	0x831464	github.com/safing/portbase/rng.tickFeeder+0x244				/home/user/git/safing/portbase/rng/tickfeeder.go:58
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0x843bae 0x8079d3 0x806f77 0x806d6a 0x4ea1a1
#	0x843bad	github.com/safing/portbase/api.(*DatabaseAPI).writer+0x12d		/home/user/git/safing/portbase/api/database.go:221
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x806f76	github.com/safing/portbase/modules.(*Module).RunWorker+0x76		/home/user/git/safing/portbase/modules/worker.go:52
#	0x806d69	github.com/safing/portbase/modules.(*Module).StartWorker.func1+0x49	/home/user/git/safing/portbase/modules/worker.go:27

1 @ 0x4bd216 0x4ccd5c 0x85bebe 0xd07b99 0x4bce7e 0x4ea1a1
#	0x85bebd	github.com/safing/portbase/run.Run+0x27d	/home/user/git/safing/portbase/run/main.go:76
#	0xd07b98	main.main+0xf8					/home/user/git/safing/portmaster/cmds/portmaster-core/main.go:36
#	0x4bce7d	runtime.main+0x1fd				/usr/local/go/src/runtime/proc.go:250

1 @ 0x4bd216 0x4ccd5c 0x982865 0x8079d3 0x8073b6 0x4ea1a1
#	0x982864	github.com/safing/portbase/notifications.cleaner+0xe4			/home/user/git/safing/portbase/notifications/cleaner.go:13
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0x99f0dd 0x8079d3 0x8073b6 0x4ea1a1
#	0x99f0dc	github.com/safing/portmaster/intel/geoip.(*updateWorker).run+0x17c	/home/user/git/safing/portmaster/intel/geoip/database.go:173
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0x9a517e 0x8079d3 0x8073b6 0x4ea1a1
#	0x9a517d	github.com/safing/portmaster/netenv.monitorNetworkChanges+0x11d		/home/user/git/safing/portmaster/netenv/network-change.go:48
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0x9a64c6 0x8079d3 0x8073b6 0x4ea1a1
#	0x9a64c5	github.com/safing/portmaster/netenv.monitorOnlineStatus+0xe5		/home/user/git/safing/portmaster/netenv/online-status.go:357
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0x9b5745 0x8079d3 0x806f77 0x806d6a 0x4ea1a1
#	0x9b5744	github.com/safing/portmaster/status.autoPilot+0x84			/home/user/git/safing/portmaster/status/autopilot.go:16
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x806f76	github.com/safing/portbase/modules.(*Module).RunWorker+0x76		/home/user/git/safing/portbase/modules/worker.go:52
#	0x806d69	github.com/safing/portbase/modules.(*Module).StartWorker.func1+0x49	/home/user/git/safing/portbase/modules/worker.go:27

1 @ 0x4bd216 0x4ccd5c 0xa8ce2b 0x8079d3 0x8073b6 0x4ea1a1
#	0xa8ce2a	github.com/safing/portmaster/profile.cleanActiveProfiles+0xaa		/home/user/git/safing/portmaster/profile/active.go:58
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0xa91d4e 0x8079d3 0x8073b6 0x4ea1a1
#	0xa91d4d	github.com/safing/portmaster/profile.startProfileUpdateChecker.func1+0x1ed	/home/user/git/safing/portmaster/profile/database.go:54
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92			/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135		/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0xaae726 0xaae4a7 0x8079d3 0x8073b6 0x4ea1a1
#	0xaae725	github.com/safing/portmaster/resolver.handleMDNSMessages+0xc5		/home/user/git/safing/portmaster/resolver/resolver-mdns.go:152
#	0xaae4a6	github.com/safing/portmaster/resolver.listenToMDNS.func9+0x26		/home/user/git/safing/portmaster/resolver/resolver-mdns.go:142
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0xad8cc5 0x8079d3 0x8073b6 0x4ea1a1
#	0xad8cc4	github.com/safing/portmaster/network.connectionCleaner+0xa4		/home/user/git/safing/portmaster/network/clean.go:25
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0xade166 0x8079d3 0x8073b6 0x4ea1a1
#	0xade165	github.com/safing/portmaster/network.openDNSRequestWriter+0xe5		/home/user/git/safing/portmaster/network/dns.go:91
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0xcda42e 0xcdc5d8 0x8079d3 0x8073b6 0x4ea1a1
#	0xcda42d	github.com/safing/portmaster/netquery.(*Manager).HandleFeed+0xcd	/home/user/git/safing/portmaster/netquery/manager.go:82
#	0xcdc5d7	github.com/safing/portmaster/netquery.(*module).start.func2+0x37	/home/user/git/safing/portmaster/netquery/module_api.go:131
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0xcdc41a 0x8079d3 0x8073b6 0x4ea1a1
#	0xcdc419	github.com/safing/portmaster/netquery.(*module).start.func3+0xb9	/home/user/git/safing/portmaster/netquery/module_api.go:137
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0xcdc86b 0x8079d3 0x8073b6 0x4ea1a1
#	0xcdc86a	github.com/safing/portmaster/netquery.(*module).start.func1+0x24a	/home/user/git/safing/portmaster/netquery/module_api.go:110
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x8073b5	github.com/safing/portbase/modules.(*Module).runServiceWorker+0x135	/home/user/git/safing/portbase/modules/worker.go:83

1 @ 0x4bd216 0x4ccd5c 0xcf9f0d 0x8079d3 0x806f77 0x806d6a 0x4ea1a1
#	0xcf9f0c	github.com/safing/portmaster/firewall.packetHandler+0xac		/home/user/git/safing/portmaster/firewall/interception.go:688
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x806f76	github.com/safing/portbase/modules.(*Module).RunWorker+0x76		/home/user/git/safing/portbase/modules/worker.go:52
#	0x806d69	github.com/safing/portbase/modules.(*Module).StartWorker.func1+0x49	/home/user/git/safing/portbase/modules/worker.go:27

1 @ 0x4bd216 0x4ccd5c 0xcfa1c6 0x8079d3 0x806f77 0x806d6a 0x4ea1a1
#	0xcfa1c5	github.com/safing/portmaster/firewall.statLogger+0x1c5			/home/user/git/safing/portmaster/firewall/interception.go:702
#	0x8079d2	github.com/safing/portbase/modules.(*Module).runWorker+0x92		/home/user/git/safing/portbase/modules/worker.go:128
#	0x806f76	github.com/safing/portbase/modules.(*Module).RunWorker+0x76		/home/user/git/safing/portbase/modules/worker.go:52
#	0x806d69	github.com/safing/portbase/modules.(*Module).StartWorker.func1+0x49	/home/user/git/safing/portbase/modules/worker.go:27


```

</details>

**Comment by Raphty** [comment]
you removed all the dns settings from within Portmaster, I guess this is why stuff is breaking down

**Comment by working-name** [comment]
@Raphty 

> I swapped system DNS servers from ISP to google, then cloudflare, then dns.watch. Does not make a difference.

System is wrong there, I meant portmaster's. And yes, I also removed everything from portmaster and let it fallback to the dns servers set for the connection in windows. That should not have my stuff breaking down, it should gracefully ask the system resolver. 

By the way this is after multiple windows reboots, and portmaster kills and shutdowns and so on. It's just bug ridden this version, in fact the past 2 versions or so have been misbehaving.

**Comment by working-name** [comment]
@Raphty this bot sucks. What were you waiting on from me?

**Comment by dhaavi** [comment]
Suction depends on your point of view: From our side, it pushes things forward.

This issue seems to have been mislabeled. Reopening.

**Comment by working-name** [comment]
Well if this feeble report doesn't help point you to an issue, then yeah, sure, let's close it.

From my side, it looks suspiciously like a 🖕 though.

**Comment by dhaavi** [comment]
![image](https://user-images.githubusercontent.com/18184155/225255703-14800201-bfa6-4b13-bd96-b140a4fcc3bb.png)

Pro Tip: Don't swear. Be respectful.
